gym_maze/maze.py

```diff
@@ -1,7 +1,13 @@
+import random
+
 PATH_MAPPING = 0
 WALL_MAPPING = 1
 REWARD_MAPPING = 9
 
+STATE_ONE_STEP = 0
```

Comment:
What does those states mean?

Reply:
They describe states of the goal generator. They are not really used right now, mostly for debugging or future use.